unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc

```diff
@@ -1422,7 +1505,8 @@ creating in thee) in the way of his quickening and renewing life.
 [.inline]
 === Some _Questions_ answered concerning the Spirit of _Christ,_ and the spirit of the Scribes and Pharisees
 
-Question 1. WHAT kind of spirit was the spirit of the Scribes and Pharisees?
+[.discourse-part]
+_Question 1._ WHAT kind of spirit was the spirit of the Scribes and Pharisees?
```

Comment:
can you downcase the `WHAT` here?

Reply:
Also, below, on line 1519, can you change it to this:

```ASCIIDOC
[.discourse-part]
_Question 2._ What kind of Spirit was the Spirit of Christ?
```

Reply:
And also do the same with line 1551, move the 5 up to the line before, so it reads:

```ASCIIDOC
[.discourse-part]
_Question 5._ What did Christ judge of the Scribes and Pharisees according
```